src/components/MentorCard/style.ts

```diff
@@ -5,11 +5,12 @@ export const WorkerCardContainer = styled.div`
 `;
 
 export const WorkerInfoHead = styled.div`
-  font-size: 1.125rem;
-  font-weight: 600;
+  font-size: ${({ theme }) => theme.typo.body1.fontSize};
+  font-weight: ${({ theme }) => theme.typo.body1.fontWeight};
+  line-height: ${({ theme }) => theme.typo.body1.lineHeight};
```

Comment:
```suggestion
  ${({ theme }) => theme.typo.body1};
```

조금 개선해보자면 일일이 가져올 필요없이 이렇게만 해도 적용됩니다.

Reply:
다음부터는 알려주신 방법 잘 사용해보겠습니다